gate-core/src/main/java/com/netflix/spinnaker/gate/services/internal/ClouddriverService.java

```diff
@@ -100,6 +100,10 @@ Map getTargetServerGroup(
       @Query("onlyEnabled") Boolean onlyEnabled,
       @Query("validateOldest") Boolean validateOldest);
 
+  @Headers("Accept: application/json")
+  @GET("/applications/{name}/rawResources")
+  List<Map> getApplicationRawResources(@Path("name") String appName);
```

Comment:
Can this be a more specific `Map`?  (Even just `Map<String, Object>`?). (And same for the other maps added in this PR.)

Reply:
Not sure I can go further than `Map<String, Object>`. Gate doesn't really know about what Clouddriver returns

Reply:
Yeah, that's usually as strongly-typed as we can get in `gate` 👍 .